hedera-node/src/main/java/com/hedera/services/context/MutableStateChildren.java

```diff
@@ -207,16 +193,22 @@ public RecordsRunningHashLeaf runningHashLeaf() {
 		return refRunningHashLeaf;
 	}
 
-	public void setRunningHashLeaf(RecordsRunningHashLeaf runningHashLeaf) {
-		this.runningHashLeaf = new WeakReference<>(runningHashLeaf);
+	@Override
+	public FCHashMap<ByteString, EntityNum> aliases() {
+		final var refAliases = aliases.get();
+		Objects.requireNonNull(refAliases);
```

Comment:
A nifty trick I like using, `Objects.requireNonNull` actually returns the value, so you can make this a one liner.

`final var refAliases = Objects.requireNonNull(refAliases);`